ports.ts

```diff
@@ -8,7 +8,12 @@ const app = Elm.Main.init({
 
 app.ports.sendVote.subscribe((vote: PublicVote | AnonVote) => {
     console.log("sendVote from elm: ", vote)
-    send({ ...vote, user_id: "whoami", type: "vote" });
+    send({ ...vote, user_id: vote.user_id, type: "vote" });
+});
+
+app.ports.sendLogin.subscribe((login: Login) => {
+    console.log("sendLogin from elm: ", login)
+    send({ ...login, type: "login" });
```

Comment:
Should we consider moving `type` over to Elm and have a single send port like we did for received ws messages?

Reply:
Sounds good.